ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json

```diff
@@ -2014,5 +2014,12 @@
     "CaseFieldID": "manageDocumentsRespondentStatementsHeading",
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "submissionRequiredFieldsInfo",
+    "UserRole": "caseworker-publiclaw-courtadmin",
```

Comment:
I think only `[EPSMANAGING]`, `[LASOLICITOR]`, `[LAMANAGING]` need access to these fields. They only need to be visible in the open state